checkClock/app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,274 @@
+// Place all the styles related to the Dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+@import 'variables.scss';
+@import 'clearStyles.scss';
+main{
+  display: flex;
+  justify-content: center;
+  align-items: start;
+  height: 100vh;
+  font-family: 'Poppins', sans-serif;
```

Comment:
For declare your font -family may can use a specific variable, declare it in you `variables.scss` file

Reply:
Fixed.We forgot to add this variable name but we already made this change.